stdlib/public/runtime/Float16Support.cpp

```diff
@@ -29,8 +29,7 @@
 
 // Android NDK <r21 do not provide `__aeabi_d2h` in the compiler runtime,
 // provide shims in that case.
-#if (defined(__ANDROID__) && defined(__ARM_ARCH_7A__) && defined(__ARM_EABI__)) || \
-  ((defined(__i386__) || defined(__i686__) || defined(__x86_64__)) && !defined(__APPLE__) && !defined(__linux__))
+#if (defined(__ANDROID__) && defined(__ARM_ARCH_7A__) && defined(__ARM_EABI__))
```

Comment:
Would this break this on Win32 and some other 32-bit x86 platforms? I don't build or use i686 anymore, so don't really care myself, but may be something to watch out for.

Reply:
I don't think that this will break the Windows x86 builds if we were to get the x86_64 builds complete.  The CPU is still the same, it is just the bitness that changes.  It will however be a bit messier codegen due to the calling convention.